sys/newlib/Makefile.include

```diff
@@ -69,4 +69,4 @@ ifeq (1,$(USE_NEWLIB_NANO))
 endif
 
 # Newlib includes should go after GCC includes.
-export INCLUDES += $(NEWLIB_INCLUDES)
+export INCLUDES := $(NEWLIB_INCLUDES) $(INCLUDES)
```

Comment:
just (re)read your description and you _want_ to have newlib before other includes, hence please adapt comment